compiler/rustc_metadata/src/native_libs.rs

```diff
@@ -119,7 +119,7 @@ impl<'tcx> Collector<'tcx> {
             let mut kind = None;
             let mut modifiers = None;
             let mut cfg = None;
-            let mut wasm_import_module = None;
+            let mut wasm_import = None;
```

Comment:
```suggestion
            let mut wasm_import_module = None;
```
Could you keep the naming matching link arguments in source code.